lib/api/include/atomic_apply_metadata_operations.h

```diff
@@ -18,6 +18,7 @@ extern "C" {
 /// \p json_input must have the following JSON structure:
 /// \code{.js}
 /// {
+///   "as_admin": boolean,
```

Comment:
is there a property name we want to standardize on for signaling **execute as administrator**?

suggestions:
- `as_admin`
- `execute_as_admin`
- `admin_mode`
- `enable_admin_mode`
- `elevate_privileges`
- `run_as_admin`
- `admin`

any others?

Reply:
our docs / help text mention 'admin mode' in a few places.

that seems... workable.

Reply:
done